models/public/hybrid-cs-model-mri/README.md

```diff
@@ -0,0 +1,49 @@
+# hybrid-cs-model-mri
+
+## Use Case and High-Level Description
+
+["A Hybrid Frequency-domain/Image-domain Deep
+Network for Magnetic Resonance Image
+Reconstruction"](https://arxiv.org/pdf/1810.12473.pdf) network. See original [project](https://github.com/rmsouza01/Hybrid-CS-Model-MRI) for the details.
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Image inpainting                          |
```

Comment:
If compare our image inpainting network and this one, they do different things (first restore some masked region, used for objects removing, second is for cleaning noisy MRI data). It may confuse users, who do not read papers and have no deep knowledge of models. Could you please extend model description to make it clear, self-sufficient to understand what this model is without reading the article (e.g. put high-level information about model architecture and use case). 

BTW, As I know, it is big topic for this release to add medical models to OMZ, possibly we also can emphasize it in model type, e.g. something like medical image restoration instead inpainting?

Reply:
@eaidova, actually, it's an inpainting model but just in different modality which is visualized as noisy grayscale image.

Reply:
@dkurt I already read, thanks :)